godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -1,3 +1,21 @@
 plugins {
     kotlin("multiplatform")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Do we need this here? We are not targeting the JVM so I guess this is not required?

Reply:
We had once the problem that a user build the lib locally with java 13 because nothing was set here, but in his game he specified java 8 in the IDE which lead the game to not be able to compile.
This solved it in that case.
Do you have a better idea to enforce java 8 compatibility when built locally?

Reply:
That's super weird. This setting should only affect a jvm compilation which in this case we don't have any. Is there a ticket somewhere I could look at? Or was it in discord?

Reply:
It was on discord. But maybe you are right. Because we had in the old binding a jvm target as the annotations were a multiplatform module which had a jvm target for the entry generator.
But we will not have that so i guess i can remove it :thinking:

Reply:
Let's add it later on once we encounter it again :)